### ISSUE TITLE
Remove "window.__statics = __dirname" when nodeIntegration = false;

### ISSUE THREAD

**Issue by newCodeRunner** [quasarframework/quasar#7580]
**Describe the bug**
When Node Integration is set to false, Electron Renderer Process does not have access to "__dirname". Hence, the index.html gives error at this inline script tag, when the Electron window loads the index.html.
```
<script>window.__statics = __dirname</script>
```

Solution:
 Let users set a preload and handle statics themselves, or better still add a preload by default for electron (when nodeIntegration is false _or not - this way if user disables in-line scripts in content-security-policy when displaying remote content, there wont be another content security error either_), that get's this statics directory at runtime. E.g in electron-preload file:
```
window.__statics = __dirname
``` 

**To Reproduce**
1. Create a Quasar Project
2. Set nodeIntegration to false in quasar.conf.js
3. Build the app using Electron-Builder
4. Run the app, and check console.

**Platform (please complete the following information):**
OS: win 10.0.17134
Electron: 9.1.2
Electron-Builder: 22.8.0

**Comment by newCodeRunner** [comment]
Temporary fix:

In file
\node_modules\@quasar\app\lib\webpack\plugin.html-addons.js

remove or comment out code at lines 51-56

```
          const bodyScript = `
            window.__statics = __dirname
          `
          data.bodyTags.push(
            makeScriptTag(bodyScript)
          )
```

**Comment by rstoenescu** [comment]
Thanks for reporting. Don't know how this slipped away.
Fix will be available in "@quasar/app" v2.0.7.
